web/modules/weather_data/src/Service/WeatherAlertTrait.php

```diff
@@ -49,8 +49,11 @@ public function getAlertsForGrid($wfo, $x, $y)
      */
     public function getAlertsForLatLon($lat, $lon)
     {
+        $lat = round($lat, 4);
+        $lon = round($lon, 4);
```

Comment:
It turns out that in some calls using lat/lon, if they have more than 4 digits of precision, we get a 301 redirect. Drupal has been helpfully following the redirect for us so it hasn't been a bug, but it does cause an extra network hop. So I opted to round these down to 4 decimal points to cut that hop out.